### PR TITLE
add footer component

### PR DESCRIPTION
**Pull request by GusBaamonde**
feat: add footer component

### CODE REVIEW

**Comment by andresmoschini** [comment]
It is fine for me, but it would be nice if someone more familiar with Doppler UI reviews it. 

CC: @IgnacioRodrigues @cbernat @stirelli @juanasas @mvillaseco @fcoronelmakingsense

**Comment by cbernat** [comment]
I see it ok.

File: assets/templates/index.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
             <div class="user-menu"> <!--open-->
               <header>
                 <span class="user-avatar--menu">GB</span>
-                <p><span class="name ng-binding">Gus Baamonde</span><span class="email ng-binding">gbaamonde@makingsense.com</span></p>
```
Comment (cbernat) [inline assets/templates/index.html]: 💯

**Pull request merged**